scrapy/extensions/postprocessing.py

```diff
@@ -22,14 +32,24 @@ def __init__(self, file: BinaryIO, feed_options: Dict[str, Any]) -> None:
                                  mtime=mtime, filename=filename)
 
     def write(self, data: bytes) -> int:
+        """Compress and write the `data` into the target file."""
         return self.gzipfile.write(data)
 
     def close(self) -> None:
+        """Close target file."""
         self.gzipfile.close()
         self.file.close()
```

Comment:
Since the purpose of the `write` and `close` methods of any plugin is described in the Custom Plugin section, there is no need to document them.

```suggestion
    def write(self, data: bytes) -> int:
        return self.gzipfile.write(data)

    def close(self) -> None:
        self.gzipfile.close()
        self.file.close()
```

If after removing their documentation they still show up when using `autoclass`, use autoclass Sphinx options to hide these 2 methods in the documentation.